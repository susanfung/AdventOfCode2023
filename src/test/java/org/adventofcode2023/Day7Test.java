package org.adventofcode2023;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

import java.util.Arrays;
import java.util.List;

class Day7Test {
    @Test
    void countAndSortCharacters() {
        String string = "32T3K";
        Approvals.verifyAll("Character Counts", Day7.countAndSortCharacters(string));
    }
}