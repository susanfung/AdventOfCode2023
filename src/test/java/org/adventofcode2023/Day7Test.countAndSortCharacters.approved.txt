Character Counts[0] = 2
Character Counts[1] = 1
Character Counts[2] = 1
Character Counts[3] = 1
